src/styles/styles.scss

```diff
@@ -139,8 +139,10 @@ html {
 
     .button1,
     .button2 {
+      display: inline-block;
       border-radius: 50px;
       border: none;
+      margin: 5px 0 5px 5px;
```

Comment:
why do we need this?  Is this the browser-default for button tags?

Reply:
This is browser default for button tags which I needed to apply for the `div` buttons too.